crates/rpc-types-eth/src/transaction/signature.rs

```diff
@@ -74,27 +74,21 @@ where
     }
 }
 
-impl TryFrom<Signature> for alloy_primitives::Signature {
+impl TryFrom<Signature> for alloy_primitives::PrimitiveSignature {
     type Error = alloy_primitives::SignatureError;
 
     fn try_from(value: Signature) -> Result<Self, Self::Error> {
         let parity = if let Some(y_parity) = value.y_parity {
-            alloy_primitives::Parity::Parity(y_parity.0)
+            y_parity.0
         } else {
-            value.v.to::<u64>().try_into()?
+            match value.v.to::<u64>() {
```

Comment:
this could panic, we should keep using try_into here

Reply:
this type and conversion are now gone with #1460 :)